src/components/Breadcrumbs/Breadcrumbs.stories.mdx

```diff
@@ -28,9 +28,8 @@ import { getCategory } from "../../utils/componentCategories";
     jest: ["Breadcrumbs.test.tsx"],
```

Comment:
The figma link points to the `Input` page.

Reply:
Good catch.